docs/references/backend/user/get-user-oauth-access-token.mdx

```diff
@@ -16,7 +16,7 @@ function getUserOauthAccessToken: (userId: string, provider: `oauth_${OAuthProvi
 | Name | Type | Description |
 | --- | --- | --- |
 | `userId` | `string` | The ID of the user to retrieve. |
-| `provider` | <code>oauth_$\{[OAuthProvider](/docs/references/javascript/types/oauth-provider)\}</code> | The OAuth provider to retrieve the access token for. |
+| `provider` | <code>oauth_$\{[OAuthProvider](/docs/references/javascript/types/oauth#o-auth-provider)\}</code> | The OAuth provider to retrieve the access token for. |
```

Comment:
```suggestion
| `provider` | `string` | The OAuth provider to retrieve the access token for. The value will match the following format: <code>oauth_$\{[OAuthProvider](/docs/references/javascript/types/oauth#o-auth-provider)\}</code> |
```

I suggest something like this because I think the type should still say `string`, but I like that you provided the template for how the value will look

Reply:
here's the function definition with the type 😳

<img width="621" alt="Screenshot 2024-03-18 at 17 48 17" src="https://github.com/clerk/clerk-docs/assets/98043211/df73b503-1219-47e4-ae75-82f76bd0826b">

what do u think

Reply:
Oh, interesting! This is a [template literal type](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html), so my suggestion is wrong.

Reply:
I have learned something new about types!